README.md

```diff
@@ -420,7 +420,8 @@ you can also use a url in the command field, if your mesos was compiled with cUR
    ],
    "schedule":"R/2014-03-08T20:00:00.000Z/PT2H",
    "environmentVariables": [
-     {"name": "FOO", "value": "BAR"}
+     {"name": "FOO", "value": "BAR"},
```

Comment:
Maybe we should use more likely examples? like `JVMOPTS="-Xmx1000m"` and `JAVA_LIBRARY_PATH=/usr/local/lib`?